src/stuf/stuf_bg/constants.py

```diff
@@ -17,6 +24,37 @@
 STUF_BG_EXPIRY_MINUTES = 5
 
 
+type DateOfBirthPrecisionType = Literal["date", "year_month", "year"]
+
+
+class NaturalPersonDetails(BaseModel):
+    bsn: str | None = None
+    first_names: str | None = None
+    initials: str | None = None
+    affixes: str | None = None
+    lastname: str | None = None
+    date_of_birth: str | None = None  # ISO - 8601
```

Comment:
is this really an ISO-8601 date string? How does that work with partial dates?

Reply:
When it's complete yes, but I guess I have to remove it since it's not always the case.